services/ocs/pkg/config/reva.go

```diff
@@ -4,3 +4,7 @@ package config
 type TokenManager struct {
 	JWTSecret string `yaml:"jwt_secret" env:"OCIS_JWT_SECRET;OCS_JWT_SECRET" desc:"The secret to mint and validate jwt tokens." introductionVersion:"pre5.0"`
 }
+
+type API struct {
+	ShowUserEmailInResults bool `yaml:"show_email_in_results" env:"OCIS_SHOW_USER_EMAIL_IN_RESULTS" desc:"Mask user email addresses in responses." introductionVersion:"5.1"`
+}
```

Comment:
Shouldn't this be 5.0.1, now that we backported it? (would need to be changed in master as well I guess)

Reply:
Thing is, it was/will be introduced in 5.1 not 5.0 it is a backport to 5.0 that does not follow the behavior it will have in 5.0 (see defaultconfig.go) so I am unsure what to actually put in there :) Let's decide tomorrow in/after the daily.